src/components/faq.js

```diff
@@ -5,6 +5,7 @@ function FAQ(props) {
   const [faq, setFaq] = useState([]);
 
   useEffect(() => {
+    document.title = "FAQ | covid19india"
     getFAQs();
   });
```

Comment:
```suggestion
  }, []);
```
We need to set inputs of useEffect as empty array. Otherwise the useEffect is run every time component is rendered and it already requests faq like a infinite loop

Reply:
@vramana: looks like this was existing bug.. Prod site is firing faq api every 2 sec.. Good catch.. I have fixed it..